src/encoded/static/components/body_map.js

```diff
@@ -620,15 +620,18 @@ class BodyMap extends React.Component {
                     <ul className="body-list-inner">
                         {Object.keys(SystemsList).map(b =>
                             <li key={b}>
-                                <button
-                                    className={`body-list-element ${checkClass(this.state.selectedOrgan, b) ? 'active' : ''}`}
+                                <span
                                     id={b}
+                                    className={`body-list-element ${checkClass(this.state.selectedOrgan, b) ? 'active' : ''}`}
+                                    role="button"
+                                    tabIndex="0"
                                     onClick={e => this.chooseOrgan(e)}
+                                    onKeyPress={e => this.chooseOrgan(e)}
                                     onMouseEnter={e => highlightOrgan(e)}
                                     onMouseLeave={unHighlightOrgan}
                                 >
                                     {b}
-                                </button>
+                                </span>
```

Comment:
The button was causing the Safari bullet issue?

Reply:
Yes, ugh. It was some built-in button style that I couldn't figure out how to remove. I tested navigation by tab and I think I managed to get everything working properly without the built-in buttons, though.